appUITests/Data/AccountsListTestData.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  AccountsListTestData.swift
+//  GoldenSampleUITests
+//
+//  Created by Backbase on 15/01/2024.
+//
+
+import Foundation
+
+struct AccountsListTestData{
+    // NOTE : Please adjust this based on the test data available
+    static let defaultAccountName: String = "Sara's Current Account"
```

Comment:
In Swift you do not need the :String type, because you are providing a default value and the system automatically knows the type from the default value. Please remove the type to conform to our Swift coding guides.